openlibrary/plugins/upstream/account.py

```diff
@@ -329,6 +374,9 @@ def POST(self):
                     verified=False,
                     retries=USERNAME_RETRIES,
                 )
+                if f.pd_request.checked:
+                    web.setcookie("rpd", "1", expires=-1)
```

Comment:
The `rpd` cookie is unnecessary here.  The `pda` cookie implies that they have clicked the checkbox.

Reply:
Updated [here](https://github.com/internetarchive/openlibrary/pull/10724/commits/e662e6aac5662b8647fbcddeb1b71e2e29a9b24e)